Core/src/org/sleuthkit/autopsy/datamodel/HashsetHits.java

```diff
@@ -370,23 +372,28 @@ protected void removeNotify() {
 
         @Override
         protected boolean createKeys(List<Long> list) {
-            list.addAll(hashsetResults.getArtifactIds(hashsetName));
+ 
+            if (skCase == null) {
+               return true;
+            }
+            
+            hashsetResults.getArtifactIds(hashsetName).forEach((id) -> {
+                try {
+                    BlackboardArtifact art = skCase.getBlackboardArtifact(id);
+                    
+                    artifactHits.put(id, art);
+                    list.add(id);
+                } catch (TskException ex) {
```

Comment:
Since this is a case database error, will you please change the log level to SEVERE instead of WARNING, while we are in this code?